autoblocks/_impl/vendor/openai/patch.py

```diff
@@ -13,7 +13,7 @@
 )
 
 
-def _mask_request_kwargs(kwargs: Dict) -> Dict:
+def mask_request_kwargs(kwargs: Dict) -> Dict:
```

Comment:
this doesn't really need to be hidden via a leading underscore since it's already in the `_impl/` folder and we only "export" `trace_openai` here: https://github.com/autoblocksai/python-sdk/blob/main/autoblocks/vendor/openai.py